infrastructure/src/main/java/org/wso2/testgrid/infrastructure/providers/AWSProvider.java

```diff
@@ -137,7 +134,7 @@ public void init(TestPlan testPlan) throws TestGridInfrastructureException {
             Path limitsYamlPath = Paths.get(TestGridUtil.getTestGridHomePath(), TestGridConstants.AWS_LIMITS_YAML);
             List<AWSResourceLimit> awsResourceLimits = awsResourceManager.populateInitialResourceLimits(limitsYamlPath);
             if (awsResourceLimits == null || awsResourceLimits.isEmpty()) {
-                logger.warn("Not using current AWS resource limits. awsLimits.yaml may be missing.");
+                logger.warn("Could not populate AWS resource limits. ");
```

Comment:
It seems there are git conflicts that was not resolved properly. Apparently we are reverting some commits done by other devs. :)